### PR TITLE
Features/react pdf viewer

### PR DESCRIPTION
**Pull request by JLC6290**
# Description

Fixes #

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "labs-spa-starter",
+  "name": "story-squad",
```
Comment (Schrese) [inline package.json:2]: Good idea changing this!

**Review by bramirez96 — APPROVED** [review]
Looks great!

**Review by Schrese — APPROVED** [review]
Just missing a description in the pull request checklist

**Review by SandraCoburn — APPROVED** [review]
Looks great!

**Review by Nick-Ohman — APPROVED** [review]
Flawless! Looks great!

**Pull request merged**